.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules/
-svg/output/
```

Comment:
Yep so:

* `build` is now simply `react`([ref](https://github.com/WordPress/dashicons/pull/197/files#diff-35b4a816e0441e6a375cd925af50752cR116))
* `svg/output` isn't used anywhere afaik, so just cleaned up